src/Compilers/Core/CodeAnalysisTest/PEWriter/BlobTests.cs

```diff
@@ -13,6 +15,8 @@
 
 namespace Microsoft.CodeAnalysis.UnitTests.PEWriter
 {
+    using Roslyn.Reflection;
```

Comment:
This is actually intentional -- usings inside the namespace take precedence over usings outside. These will be removed once we move the code to SRM.